docs/lit/examples/6-dl.jl

```diff
@@ -40,6 +40,7 @@ using Distributions: Poisson
 using BSON: @load, @save
 import BSON # load
 using InteractiveUtils: versioninfo
+using Downloads: download
```

Comment:
good suggestion - i didn't know about this.
but then you must use `Downloads.download` below